### ISSUE TITLE
Board Review: Azure Communication Services (SPOOL) Calling - Custom Teams Endpoint (JS)

### ISSUE THREAD

**Issue by garchiro7** [Azure/azure-sdk#4263]
Service team responsible for the client library.
**ACS- Calling team**

Contact email (if service team, provide PM and Dev Lead).

Anand Srinivasan (ansrin@microsoft.com)
Xiaoli Xiang (xiaoli.xiang@microsoft.com)

Name of the client library.

**Calling**

Languages for this review.
**JS**

Artifacts per language.
[**APIVIEW**](https://apiview.dev/Assemblies/Review/ab8311aae28040eb90c960d0fa6f9e41?revisionId=e8ff07ce637745d7b568dcc379564f57&diffRevisionId=e21bf7bc543446b684788951337775da&doc=False&diffOnly=False)

[Champion scenarios.](https://microsoft-my.sharepoint-df.com/:w:/p/xixian/EdHKi599W5hFjK_PflDY_9YBlLb8ZoaddOXwazw_MpB_Kw?e=HrAQF2)

**Comment by tg-msft** [comment]
[Recording](https://msit.microsoftstream.com/video/ff7b0840-98dc-b478-e0c8-f1ece778064f) (MS INTERNAL ONLY)

**Comment by xixian73** [comment]
Here's the meeting summary:

1. We reached agreement on the approach we are going to take for splitting ACS and CTE
2. Regarding naming, we agreed on keeping the teams as prefix for objects. Eg. TeamsCallAgent, TeamsCall, TeamsIncomingCall, etc.
3. I will start offline chat thread with Jeff, ⁠Jose, and ⁠Filippos about hiding the Base from exposing to developers
4. We did initial assessment on Native for the same approach. Should be technically achievable
5. I will continue to work with ⁠Jose to get final signoff in apiview
